tests/system/action/base.py

```diff
@@ -347,7 +366,7 @@ def base_permission_test(
             self.set_models(models)
         response = self.request(action, action_data)
         if permission:
-            self.assert_status_code(response, 200)
+            self.assertIn(response.status_code, (200, 202))
```

Comment:
You can probably revert this with the new timout of -1

Reply:
done